Marlin/src/HAL/HAL_STM32F1/watchdog_Stm32f1.h

```diff
@@ -39,6 +45,11 @@ void watchdog_init();
 
 // Reset watchdog. MUST be called at least every 4 seconds after the
 // first watchdog_init or STM32F1 will reset.
-inline void watchdog_reset() { iwdg_feed(); }
+inline void watchdog_reset() {
+  #if PIN_EXISTS(LED)
+    TOGGLE(LED_PIN);  // heart beat indicator
```

Comment:
I'm not sure if we want this always. Depends on whether users want control over their `LED_PIN` for other uses.

Reply:
Is there a configuration option somewhere to decide what to do with the LED? I did not see any, and copied that bit from the LPC HAL, so the LED would provide a heart bit.

Reply:
> Is there a configuration option somewhere to decide what to do with the LED?

No, there is not a config option for it. I'm sure users will ask for one if they need it someday.

> copied that bit from the LPC HAL, so the LED would provide a heart bit.

That's fine, if we're setting a precedent.